egs/multi_lang/s5/run-4-anydecode-langs.sh

```diff
@@ -0,0 +1,583 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+
+dir=dev10h.pem
+kind=
+data_only=false
+fast_path=true
+skip_kws=false
+skip_stt=false
+skip_scoring=
+extra_kws=true
+vocab_kws=false
+tri5_only=false
+use_sep_init_layer=false
+use_pitch=true
+use_entropy=false
+use_ivector=false
+use_raw_wave_feats=false
+pitch_conf=conf/pitch.conf
+wip=0.5
+decode_stage=-1
+nnet3_dir=nnet3/tdnn_sp
+is_rnn=false
+use_raw_wave_feats=false
+extra_left_context=0
+extra_right_context=0
+frames_per_chunk=0
+wav_input=segmented_wav.scp
```

Comment:
Are you committing raw wave feats here? Also are you checking in the entropy features? If you do not have source files for that, you might have to remove these from the scripts.